dataplane/ebpf/src/utils.rs

```diff
@@ -51,6 +61,20 @@ pub fn ip_from_int(ip: u32) -> [u8; 4] {
     addr
 }
 
+//Return logging string
+pub fn stringify_destip_for_logging(daddr: u32, protocol_code: u8, port: u16) -> String {
```

Comment:
Apologies but as am newbie :( i am not sure how do I import `String` and `format` in this crate. As root module specified `no_std` i think i can't use this functions and core module seems to only provide `core::str`

Reply:
So unfortunately we'll need to work with raw bytes i,e `[u8]` for any code actually being loaded into the kernel (which includes these util functions) since we can't use any types from the standard lib. 

Now that I dug a bit deeper though... Can you try out this aya-log feature https://github.com/aya-rs/aya/commit/83ec27f06b6859f455f2b2baf985b8fd3fb4adc5? 

Here's an example of it being used -> https://github.com/vadorovsky/aya-examples/blob/main/tc/tc-ebpf/src/main.rs#L40

That might allow us to remove this alltogether

Reply:
cc @shaneutt